src/test/java/org/sonar/plugins/github/PullRequestIssuePostJobTest.java

```diff
@@ -167,6 +173,99 @@ public void testPullRequestAnalysisWithNewIssues() {
   }
 
   @Test
+  public void testSortIssues() {
+    ArgumentCaptor<String> commentCaptor = forClass(String.class);
+
```

Comment:
This is a matter of personal choice. Some people hates mock and avoid them at all cost. I tend to prefer using mock especially for big interface when you only use a small part of it. It make it more obvious what are your dependencies.